azureLogIngestion/opentelemetry/nrOtelTracerProvider.d.ts

```diff
@@ -0,0 +1,83 @@
+import * as api from '@opentelemetry/api';
+import { InstrumentationLibrary } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
+import { BasicTracerProvider } from '@opentelemetry/tracing/build/src/BasicTracerProvider';
+import { SpanLimits, TracerConfig } from '@opentelemetry/tracing/build/src//types';
+/**
+ * This class represents a basic tracer.
+ */
+export declare class NrOtelTracerProvider extends BasicTracerProvider {
+  private _tracerProvider;
+  private readonly _sampler;
+  private readonly _spanLimits;
+  private readonly _idGenerator;
+  private _resource: Resource
+  // @ts-ignore
+  public get resource(): Resource {
+        return this._resource;
```

Comment:
@mrickard You don't include  an implementation in a definition. Might be the source of your error.